Taskfile.yml

```diff
@@ -169,7 +169,7 @@ tasks:
   ##
   client:test:
     desc: Unit test Client code
-    dir: ./client
+    dir: ./client/go
```

Comment:
lets keep the client golanf code as is and move only the python code to 'sdk/python/client'. i would only suggest to keep the location of golang code as is